src/constants.ts

```diff
@@ -20,3 +20,14 @@ export const SUGGEST_TEMPLATE_LINK =
  * Splitbee events
  */
 export const SPLITBEE_HERO_SUGGEST_TEMPLATE = 'hero_click_suggest_template';
+
+/**
+ * SEO constants
+ */
+export const SEO_TITLE = 'Chakra Templates';
+export const SEO_DESCRIPTION =
+  'A growing open-source collection of hand-crafted Chakra UI templates ready to drop into your React project.';
+export const SEO_KEYWORDS =
+  'chakra ui, ui templates, web templates, web design, react templates, react ui templates, chakra ui templates, chakra templates';
+export const SEO_OG_IMAGE = 'https://chakra-templates.dev/favicon.png';
+export const SEO_TWITTER_IMAGE = 'https://chakra-templates.dev/favicon.png';
```

Comment:
We should design some cool OG/Twitter images in the future :)

Reply:
That's exactly what I was planning to do! 😁